File: tennis/application/Models/TournamentRegistration.py
# from django.db import models
# from ..Models import TennisPlayer, Tournament
#
# class TournamentRegistration(models.Model):
#     tr_registration_date = models.DateField()
#     tr_last_year_performance = models.CharField(max_length=100)
#     tr_player = models.ForeignKey(TennisPlayer, on_delete=models.CASCADE, null=False)
#     tr_tournament = models.ForeignKey(Tournament, on_delete=models.CASCADE, null=False)

File: react-frontend/src/models/TournamentRegistration.ts
import { TennisPlayer } from "./TennisPlayer"
import { Tournament } from "./Tournament"

export interface TournamentRegistration {
    id: number;
    tr_registration_date: string;
    tr_last_year_performance: string;
    tr_player: TennisPlayer;
    tr_tournament: Tournament; 
}